moto/dynamodbstreams/responses.py

```diff
@@ -23,8 +23,13 @@ def get_shard_iterator(self):
         arn = self._get_param('StreamArn')
         shard_id = self._get_param('ShardId')
         shard_iterator_type = self._get_param('ShardIteratorType')
+        sequence_number = self._get_param('SequenceNumber')
+        # according to documentation sequence_number param should be string
+        if isinstance(sequence_number, "".__class__):
```

Comment:
Can we use [six.string_types](https://six.readthedocs.io/#six.string_types) here?